content/topics/git/git-introduction/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+content_type: topic
+title: "Introduction to Git and GitHub Part: 2"
+ready: True
+---
+
+## Navigating Through Git Commands
```

Comment:
Starting with a level 2 heading is a little strange.  Maybe have a little intro.

Eg: In this topic, we'll cover a few common commands, and spend some time understanding its distributed nature. 

Not critical but jumping straight from one heading to the next without any context can be a bit jarring.